src/jwplayer/types.d.ts

```diff
@@ -155,6 +155,22 @@ export interface OnPlaylistItemEventData {
 	item: PlaylistItem;
 }
 
+interface CaptionTrack {
+	id: number | string;
+	label: string;
+	language?: string;
+}
+
+export type CaptionsList = CaptionTrack[];
+
+export interface OnCaptionsEventData {
+	track: number;
+	tracks: CaptionsList;
+	type: string;
+}
+
+type onCaptionsChangedEventData = number;
```

Comment:
Is this used anywhere? I couldn't find a reference

Reply:
It was at one point, but I don't think it is any longer. I will remove it.